tests/clients/janus/test_janus_api_client.py

```diff
@@ -0,0 +1,56 @@
+"""Module to test Janus API client."""
+
+from http import HTTPStatus
+
+import pytest
+import requests
+from pytest_mock import MockFixture
+
+from cg.clients.janus.api import JanusAPIClient
+from cg.clients.janus.dto.create_qc_metrics_request import CreateQCMetricsRequest
+from cg.clients.janus.exceptions import JanusClientError
+
+
+def test_qc_metrics_successful(
+    mocker: MockFixture,
```

Comment:
I resolved to MockFixture here since this package is already installed. "requests_mock" would introduce a new dependency

Reply:
I think it is ok to introduce since it just is a dev depdency

Reply:
Let's discuss on the standup monday then I can change it if needed after.